infrastructure/terraform/README.md

```diff
@@ -0,0 +1,29 @@
+# OpenTofu based deployment
+
+## Requirements
+
+- OpenTofu
+- GCloud CLI
+- GCP Project
+
+## Bootstrap
+
+To bootstrap the account
+
+## User access
+
+To allow a user to execute the GKE terraform, add it to the [user-access module](./user-access/README.md).
+
+**WARNING:** This gives a lot of permission in the account that are not only restricted to the GKE cluster.
+
+### Argo CD deployment
```

Comment:
I am missing the step by step guide about how to update infrastructure using this solution.